bip-0388.mediawiki

```diff
@@ -14,7 +14,9 @@
 
 == Abstract ==
 
-Wallet policies build on top of output script descriptors to represent the types of descriptors that are typically used to represent "accounts" in a software wallet, or a hardware signing device, in a compact, reviewable way. A wallet policy always represents descriptors which produce all the receive and change addresses that are logically part of the same account.
+Software wallets and hardware signing devices sequester wallet uses into logically separate "accounts".
```

Comment:
Good point on 'sequester'; I used 'partition the funds' in the rewrite, preferring to refer specifically to "funds" rather than the more generic "wallet uses".

I spent quite some time trying to reformulate the sentence, never feeling fully happy with it. I think the reason is that the while the goal of the accounts (= keeping funds partitioned) was stated, the practical effect (= designing a UX that abstracts away UTXOs, matching the user's expectation) was not stated at all in the entire abstract. So in c2655e0 I added a new sentence to this effect.